kimchi/wasm/src/srs.rs

```diff
@@ -133,14 +133,13 @@ macro_rules! impl_srs {
                 let x_domain = EvaluationDomain::<$F>::new(domain_size as usize).ok_or_else(|| {
                     JsValue::from_str("caml_pasta_fp_urs_lagrange_commitment")
                 })?;
-
-                {
+                crate::rayon::run_in_pool(|| {
```

Comment:
Is this important for part of this fix? If it's unrelated, can you follow up in another PR?

Reply:
it's related insofar as it fixes a snarkyjs test in mina CI

Reply:
Can you describe what you're fixing here and why this fixes it? 

It looks like this runs in direct contradiction to the comment below. We probably need a mutex or similar on the SRS to do this safely.

Reply:
I know, I was worrying about the comment as well.. It fixes this error when running `snarkyjs/tests/integration/inductive-proofs.js`, which failed because it tried to use rayon in this function and there was no thread pool

Reply:
is the function expensive? otherwise we could run it single-threaded somehow

Reply:
> is the function expensive?

Yes.

> I know, I was worrying about the comment as well..

I suppose it's good enough for now, but we should switch to using a mutex ASAP. Not in this PR though.